aten/src/TH/THTensorApply.h

```diff
@@ -461,7 +460,7 @@
       } \
     } else {\
       PRAGMA(simd) \
-      PRAGMA( omp parallel for if (SIZE > TH_OMP_OVERHEAD_THRESHOLD_OMP) )  \
+      PRAGMA( omp parallel for if (SIZE > OMP_THRESHOLD * 10) )  \
```

Comment:
The magical `* 10` are weird. Why can't we just increase the threshold?

Reply:
@apaszke we can increase the threshold directly. But we just want to follow the exsiting code. eg. light weight operations like [left shift ](https://github.com/pytorch/pytorch/blob/master/aten/src/TH/generic/THTensorMath.c#L924).

Reply:
The input parameter OMP_THRESHOLD is for discontiguous tensors' operation. As for contiguous tensors' operation, the threshold is usually 10 times larger than discontiguous ones.